src/main/java/uk/gov/justice/hmpps/prison/api/resource/BookingResource.java

```diff
@@ -879,7 +879,7 @@ public AdjudicationSummary getAdjudicationSummary(@PathVariable("bookingId") @Ap
         @RequestParam(value = "fromDate", required = false) @org.springframework.format.annotation.DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @ApiParam("Returned visits must be scheduled on or after this date (in YYYY-MM-DD format).") final LocalDate fromDate,
         @RequestParam(value = "toDate", required = false) @org.springframework.format.annotation.DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @ApiParam("Returned visits must be scheduled on or before this date (in YYYY-MM-DD format).") final LocalDate toDate,
         @RequestParam(value = "visitType", required = false) @ApiParam(value = "Type of visit", allowableValues = "SCON, OFFI") final String visitType,
-        @RequestParam(value = "visitStatus", required = false) @ApiParam(name = "Status of visit. code from VIS_STS domain, e.g: Cancelled (CANC) or Scheduled (SCH)", example = "SCH") final String visitStatus,
+        @RequestParam(value = "visitStatus", required = false) @ApiParam(name = "Status of visit. code from VIS_COMPLETE domain, e.g: Cancelled (CANC) or Scheduled (SCH)", example = "SCH") final String visitStatus,
```

Comment:
Confirmed by selecting all the different values in preprod, namely
```
select VISIT_STATUS, count(VISIT_STATUS) from OMS_OWNER.OFFENDER_VISITS group by VISIT_STATUS
```
which gives
```
CANC,2026931
EXP,110
HMPOP,7374
NORM,7078102
OFFEND,29646
SCH,5263401
VDE,1684
VISITOR,66619
```
The 110 `EXP` values look like they are dodgy data from 2007, all for `ALI` prison.
Also from NOMIS:
![Screenshot from 2022-01-13 15-16-48](https://user-images.githubusercontent.com/5099053/149357460-ae50f617-2750-4b5b-8dd3-dfb67455ef67.png)
Note that `VIS_COMPLETE` values are a sub set of the `VIS_STS` values, but as a filter it makes more sense to only select on values that will be there.